prelude/bootstrap.js

```diff
@@ -565,10 +569,53 @@ function payloadFileSync(pointer) {
   // open //////////////////////////////////////////////////////////
   // ///////////////////////////////////////////////////////////////
 
+  function removeTemporaryFolderAndContent(folder) {
+    if (NODE_VERSION_MAJOR <= 14) {
+      if (NODE_VERSION_MAJOR <= 10) {
+        // folder must be empty
+        for (const f of fs.readdirSync(folder)) {
+          fs.unlinkSync(path.join(folder, f));
```

Comment:
Could it be that `f` is a directory? If so this would fail

Reply:
We are controlling what goes in this temp folder, there are only flat files by design.